From 8e5a885158f430de3ea36b1439dd8c0058ce95df Mon Sep 17 00:00:00 2001
From: Laurentiu Palcu <laurentiu.palcu@nxp.com>
Date: Fri, 22 Nov 2019 10:12:50 +0200
Subject: [PATCH 44/49] arm64: dts: imx8mq: add DCSS external oscillator
 support

The external oscillator, which is high precision, will be used when DCSS output
goes to HDMI.

Signed-off-by: Laurentiu Palcu <laurentiu.palcu@nxp.com>
---
 arch/arm64/boot/dts/freescale/imx8mq.dtsi | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/imx8mq.dtsi b/arch/arm64/boot/dts/freescale/imx8mq.dtsi
index 5a617f9ed8b5..b75252a65c44 100644
--- a/arch/arm64/boot/dts/freescale/imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/imx8mq.dtsi
@@ -1113,8 +1113,11 @@ dcss: display-controller@32e00000 {
 					 <&clk IMX8MQ_CLK_DISP_AXI_ROOT>,
 					 <&clk IMX8MQ_CLK_DISP_RTRM_ROOT>,
 					 <&clk IMX8MQ_VIDEO2_PLL_OUT>,
-					 <&clk IMX8MQ_CLK_DISP_DTRC>;
-				clock-names = "apb", "axi", "rtrm", "pix", "dtrc";
+					 <&clk IMX8MQ_CLK_DISP_DTRC>,
+					 <&clk IMX8MQ_VIDEO2_PLL1_REF_SEL>,
+					 <&clk IMX8MQ_CLK_PHY_27MHZ>;
+				clock-names = "apb", "axi", "rtrm", "pix", "dtrc", "pll_src",
+						      "pll_phy_ref";
 				assigned-clocks = <&clk IMX8MQ_CLK_DISP_AXI>,
 						  <&clk IMX8MQ_CLK_DISP_RTRM>,
 						  <&clk IMX8MQ_VIDEO2_PLL1_REF_SEL>;
-- 
2.29.2

